### PR TITLE
feat: add link to mirror node data for entities

### PR DESCRIPTION
**Pull request by Sheng-Long**
**Description**:
Add link to mirror node data for entities

**Related issue(s)**: #804 

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Comment by Sheng-Long** [comment]
Not entirely sure what's going on in the unit tests, will check it out

File: src/components/MirrorLink.vue [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+<!-- --------------------------------------------------------------------------------------------------------------- -->
+<!--                                                     TEMPLATE                                                    -->
+<!-- --------------------------------------------------------------------------------------------------------------- -->
+
+<template>
+  <a :href="`${networkUrl}api/v1/${entityUrl}/${loc}`">
+    <span class="is-numeric">Mirror Data</span>
+  </a>
+</template>
+
```
Comment (svienot) [inline src/components/MirrorLink.vue:30]: I suggest to use the following to make the link less prominent (it is for the advanced user) and also not force the user to navigate back to the explorer page:

```
  <a :href="`${networkUrl}api/v1/${entityUrl}/${loc}`" target="_blank">
      <p class="has-text-right has-text-grey h-is-property-text">Show raw data</p>
  </a>
```

File: src/components/MirrorLink.vue [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  name: "MirrorLink",
+  components: {},
+  props: {
+    network: String,
+    entityUrl: String,
+    loc: String,
+  },
+
+  setup(props, context) {
+    const networkUrl = props.network == "mainnet" ?  "https://mainnet-public.mirrornode.hedera.com/" : "https://testnet.mirrornode.hedera.com/"
```
Comment (svienot) [inline src/components/MirrorLink.vue:48]: We need to take care of previewnet as well.

File: src/components/contract/ContractResult.vue [hunk 3]
```diff
@@ -118,6 +118,9 @@
 
     <ContractResultLogs :logs="contractResult?.logs" :block-number="blockNumber" :transaction-hash="transactionHash"/>
 
+    <!--Workaround to ensure margin bottom-->
+    <div />
+
```
Comment (svienot) [inline src/components/contract/ContractResult.vue:123]: @Sheng-Long  you may rebase your branch onto main. You will get a small fix which solves such margin discrepancies at the end of the pages.
You can get rid of that workaround `<div>`

File: src/pages/TopicDetails.vue [hunk 4]
```diff
@@ -45,6 +45,10 @@
 
     </DashboardCard>
 
+    <div style="height: 24px" />
```
Comment (svienot) [inline src/pages/TopicDetails.vue:48]: You can also get rid of that `<div>` element -- see previous comment.

**Review by svienot — CHANGES_REQUESTED** [review]
See my couple comments.
Other than that, looks good. This is going to be very useful (for us, to begin with)!

**Review by svienot — APPROVED** [review]
LG

**Pull request merged**